src/cameras/map/hooks/use-calculate-lat.js

```diff
@@ -0,0 +1,7 @@
+import { useItems } from 'cameras/map/hooks/use-items'
+
+export const useCalculateLat = () => {
```

Comment:
Whenever possible, it's best to avoid abbreviations, as they can confuse the reader.

Par example, if I'm not already aware of what you want to do, I might think that `lng` stands for `language` instead of `longitude`.

Reply:
Done